gammapy/datasets/simulate.py

```diff
@@ -88,11 +88,11 @@ def sample_sources(self, dataset):
             table = self._sample_coord_time(npred, temporal_model, dataset.gti)
             if len(table) > 0:
```

Comment:
Is this case still needed?

Reply:
No, it's not. 
We would need a condition only to build the astropy table in case of zero sampled events, I think.